ontology/ntr_terms.py

```diff
@@ -580,4 +580,10 @@
         "organs": ["blood", "bodily fluid"],
         "systems": ["immune system"],
     },
+    "NTR:0000756": {
+        "cells": ["erythroid progenitor cell"],
+        "name": "Human Umbilical cord blood-Derived Erythroid Progenitor-2",
```

Comment:
In the Cellosaurus record they have HUDEP-2 as the official name and this long name as a synonym (HUDEP2 is also a synonym).